config/badges.json

```
@@ -4,9 +4,9 @@
     "Admin Candidate": "candidate",
     "Trial Admin": "trialmin",
     "Badmin": "badmin",
-    "Admin": "admin",
+    "Game Admin": "admin",
```

```suggestion
    "Admin": "admin",
    "Game Admin": "admin",
```
Keep the old ones as well